docs/optimizing-test-execution/subsetting-tests.md

```diff
@@ -210,6 +211,24 @@ The file will contain the subset of test that should be run. You can now invoke
 bundle exec rails test $(cat launchable-subset.txt)
 ```
 
+### Robot
+
+To select a meaningful subset of tests, use `--dryrun` option to list all test cases, like this:
+
+```bash
+robot --dryrun -o dryrun.xml
+launchable subset \
+  --build <build name> \
+  --target <target> \
+  robot dryrun.xml > launchable-subset.txt
+```
+
+The file will contain the subset of test that should be run. You can now invoke Robot to run exactly those tests:
+
+```bash
+bundle exec rails test $(cat launchable-subset.txt)
```

Comment:
Oh @Konboi what's the right `robot` command here?

Reply:
yes right, just moment

Reply:
I think it may be `robot $(cat launchable-subset.txt)`.

Reply:
I thought that but I ran it and failed, so I'll investigate it. 

```
$ robot . $(cat subset.txt) .
[ ERROR ] Parsing '-s' failed: File or directory to execute does not exist.

Try --help for usage information.
```

Reply:
`robot $(cat launchable-subset.txt) .` is correct.
The cause of the error was dependent on my environment.

Reply:
:pray:
https://github.com/launchableinc/cli/pull/174